portfolio/src/main/webapp/style.css

```diff
@@ -63,9 +63,16 @@ iframe {
     justify-content: center;
 }
 
-#map {
+.chart {
     padding: 1rem;
     border: 3px dashed black;
     height: 500px;
     width: 500px;
-}
\ No newline at end of file
+}
+
+.regions-container {
+    width: 100%;
+    display: grid;
+    justify-content: center;
+    padding: 1rem;
+}
```

Comment:
Add an empty line at the end for files. Useful for the programs that read files to know the end